x-pack/plugins/apm/public/components/alerting/register_apm_alerts.ts

```diff
@@ -12,7 +12,7 @@ import {
   ALERT_EVALUATION_THRESHOLD,
   ALERT_EVALUATION_VALUE,
   ALERT_SEVERITY_LEVEL,
-} from '@kbn/rule-data-utils/target/technical_field_names';
+} from '@kbn/rule-data-utils';
```

Comment:
These were targeted imports to prevent page load bundle bloat. Does the new setup not allow for this, or does it tree-shake unused exports?

Reply:
There are other places on `apm` where we already import the entire pkg entirely in the apm plugin. In any case, the new pkg setup allow for targeted imports but you have to import types and the code separately as they are also in different folders.